custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -942,6 +943,21 @@ def _prepare_docker_command(self, options, run_mode, raw_arguments):
             )
             docker_cmd_args += " -p {port}:{port}".format(port=port)
 
+        if "runtime_params_file" in options and options.runtime_params_file:
+            docker_cmd_args += ' -v "{}":{}'.format(
+                options.runtime_params_file, in_docker_runtime_parameters_file
+            )
+            DrumUtils.replace_cmd_argument_value(
+                in_docker_cmd_list,
+                ArgumentsOptions.RUNTIME_PARAMS_FILE,
+                in_docker_runtime_parameters_file,
+            )
+            # check if runtime params is provided through env vars
+            if "RUNTIME_PARAMS_FILE" in os.environ:
```

Comment:
What would be the behavior if the user provides the runtime params file both as a command line argument and as an environment variable? IMHO, the command line argument should have precedence over the environment variable and therefore, you should add an `else` clause on top of this code block.

Reply:
we should already be getting this from the `options` please test that and remove this.

Reply:
similar to the previous comment, the reliance on a string here is extremely suspicious.  and I'm pretty sure this is already done inside of the args parser.

Reply:
the command line argument has precedence over the env var, and this is already part of the argument parsing, but what I need to add is the subcommand, so this is fixed, check :
* first, check if we need to update the current cmd , if so update the cmd by `DrumUtils.replace_cmd_argument_value`
* if is provided by env var update the `-e PARAM_FILE=/path/to/file.yaml`